Examples/SampleAppObjC/Podfile

```diff
@@ -0,0 +1,13 @@
+# Uncomment the next line to define a global platform for your project
+# platform :ios, '13.0'
+
+target 'SampleAppObjC' do
+   pod 'Rudder'
+   pod 'RudderFirebase', '~> 1.1.0'
+  # Comment the next line if you don't want to use dynamic frameworks
+  use_frameworks!
+
+  # Pods for SampleAppObjC
+
+end
+
```

Comment:
Why have we created another `Podfile` in the sample app? We already have the `Podfile` at the root. Please remove this and utilise that root-level Podfile.